PROTO_tests/tests/groupby_test.py

```diff
@@ -654,6 +687,73 @@ def test_large_mean_aggregation(self):
         for m in means.to_list():
             assert np.isclose(float(a[0]), m)
 
+    #   ak.unique takes 1 pda argument and 3 booleans
+    #      However, not all 8 combinations of the booleans are needed to
+    #      cover the test space.
+    #      Combinations TTF and TFF are supersets of all other possible
+    #      combinations, so only those are tested below.
+
+    @pytest.mark.parametrize("data_type", UNIQUE_TYPES)
+    @pytest.mark.parametrize("prob_size", pytest.prob_size)
+    def test_unique(self, data_type, prob_size):
+        Jenny = pytest.seed if pytest.seed is not None else 8675309
+        T = True
+        F = False
+        np.random.seed(Jenny)
```

Comment:
since we only use `Jenny` in one place I think it makes sense to just do:
```python
np.random.seed(pytest.seed if pytest.seed is not None else 8675309)
```
Also I think it's not super necessary to alias True and False, but this is all just stylistic